[13.0][FIX] Avoid override of function passing it to l10n_es_aeat: Ticketbai / SII.

ao-landoo opened the pull request:
Cherry-pick de los commits del PR #2595

sanchonuria committed:
[FIX] l10n_es_ticketbai_batuz: Avoid override of function

etobella committed:
[IMP] l10n_es_aeat: Add standard function for getting taxes

New hook for managing changes on the function
New field for tax_amount

etobella committed:
[IMP] l10n_aeat_sii_oca: Remove function passed to l10n_es_aeat

etobella committed:
[IMP] l10n_es_aeat_sii_oca: Use quote amount

etobella committed:
[FIX] l10n_es-dua_ticketbai_batuz: use the right function

ao-landoo commented:
@etobella

ao-landoo commented:
Curioso que en la 13 está haciendo que falle un test del SII OSS

etobella commented:
Seguramente se debe a que en 14 está refactorizado :thinking: 

en cualquier caso, estamos hablando con @pedrobaeza un cambio a nivel de como se hacen los hooks, así que lo dejamos en Stand By por ahora

Commit:
[IMP] l10n_es_aeat: Hook for deprecated method

- Enables backwards compatibility and warns the user, to be removed in future migrations

hildickethan commented:
¿Se podría añadir un cherry-pick de https://github.com/OCA/l10n-spain/pull/2608?
